src/core/lender/deployer.sol

```diff
@@ -93,11 +93,18 @@ contract DistributorLike {
     function balance() public;
 }
 
-contract OperatorLike {
+contract AllowanceOperatorLike {
     function rely(address usr) public;
     function deny(address usr) public;
 }
 
+contract WhitelistOperatorLike {
+    function rely(address usr) public;
+    function deny(address usr) public;
+    function relyInvestor(address usr) public;
+    function denyInvestor(address usr) public;
+}
+
```

Comment:
I want to avoid that.

The idea is to have only an `OperatorFab`. 

You need to cast it in the test to a whitelist Operator